test/metabase/test/data/drill.clj

```diff
@@ -0,0 +1,177 @@
+(ns metabase.test.data.drill
+  (:require [clojure.java.jdbc :as jdbc]
+            [clojure.string :as s]
+            [clojure.data.csv :as csv]
+            [environ.core :refer [env]]
+            [medley.core :as m]
+            [honeysql.core :as hsql]
+            (metabase.driver [generic-sql :as sql]
+                             [drill :as drill-driver])
+            (metabase.test.data [generic-sql :as generic]
+                                [interface :as i])
+            [metabase.util :as u]
+            [metabase.util.honeysql-extensions :as hx]
+            [clojure.java.io :as io])
```

Comment:
require sorting

Reply:
Done

Reply:
it's still out of order

Reply:
If you're using emacs check out the package `clj-refactor`. If you `(setq cljr-favor-prefix-notation t)` and run the command `clr-clean-ns` it will automatically fix namespace require sorting for you. That's what myself and the other core metabase devs use